linux-create-vm.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+# Update and Upgrade apt
+
+sudo apt update
+sudo apt upgrade
+
+# Install git
+  echo "git should be installed"
+if ( which git )
+then 
+  echo "git already installed"
+else 
+  echo "installing git"
+  sudo apt install git
+fi
+
+#install nano
+  echo "nano should be installed"
+if ( which nano )
+then
+  echo "nano already installed"
+else
+  echo "installing nano"
+  sudo apt install nano 
+fi
+
+#install snap
+  echo "nano should be installed"
+if ( which snap )
+then
+  echo "snap already installed"
+else
+  echo "installing snap"
+  sudo apt install snap 
+fi
+
+# Install multipass
+  echo "installing multipass on $(uname)"
+if  ( which multipass )
+then
+  echo "multipass already installed on $(uname)"
+else
+  echo "installing multipass on $(uname)"  
+  sudo snap install multipass
+  sleep 5
+fi
+
+# Set Multipass local driver to Qemu
```

Comment:
Do you need to do this anymore since the default is Qemu?

Reply:
Probably not.

Reply:
I did not this, it must have happened recently? @stephenlauck 
`By default, Multipass on Linux uses the qemu or lxd driver (depending on the architecture). By default, Multipass on macOS uses the qemu driver. By default, Multipass on Windows uses the hyperv driver.`

Reply:
So we probably do not need it on the darwin create either? @stephenlauck